executor/aggfuncs/builder.go

```diff
@@ -312,3 +314,26 @@ func buildBitAnd(aggFuncDesc *aggregation.AggFuncDesc, ordinal int) AggFunc {
 	}
 	return &bitAndUint64{baseBitAggFunc{base}}
 }
+
+// buildVarPop builds the AggFunc implementation for function "VAR_POP".
```

Comment:
Add description for `VAR_POP`.

Reply:
Comments are written with reference to existing functions. Anything needs to be added?

Reply:
@winoros Yes, I will impl it on TIKV too.